CMakeLists.txt

```diff
@@ -30,11 +30,42 @@ file(GLOB_RECURSE TVL_GENERATOR_SOURCES CONFIGURE_DEPENDS
 )
 set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${TVL_GENERATOR_SOURCES})
 
+# check for C++20 Concepts features
+if("cxx_std_20" IN_LIST CMAKE_CXX_COMPILE_FEATURES)
+	message("Compiler supports C++20")
```

Comment:
Since the messages below ("Compiler supports c++20 and concepts / but no concepts")
already contain this information, i would remove this, cmake output is noisy enough as is.

Reply:
+1